web/app/main.js

```diff
@@ -35,8 +36,10 @@ var selectedCountryCode;
 var forceRemoteEndpoint = false;
 var customDate;
 var timelineEnabled = false;
+var reqLang = 'en';
```

Comment:
Yes it was what I did originally but because my french locale was not understood by server.js, it results in some mix between English text/French "source names"...
the server should be able to understand fr from i18n spec, not sure what is going wrong here. 
Anyway I thought the query string could be a good intermediate solution for first release before we get the text actually read and corrected by external reviewer